soapbox/general/smonn-efficiency.md

```diff
@@ -0,0 +1,59 @@
+# Efficiency as a programmer
+
+📝 By [Simon][smonn]
+
+[smonn]: https://github.com/smonn
+
+---
+
+Someone once said that a great programmer is a lazy one. The key being that they try to get as much done with as little effort as possible as opposed to not getting anything done. In other words, they are efficient. So what can you do to improve your efficiency?
+
+## TL;DR
+
+- Learn and use keyboard shortcuts
+- Write less/more code
+- Ask questions early and often
+- Sometimes you just gotta RTFM
+- Always be learning
+
+## Learn and use keyboard shortcuts
+
+This applies to any editor (or IDE). As soon as you find yourself doing something repetitive that _feels_ slow, try to find out if there's a keyboard shortcut for it. If not, can you add one?
+
+For example, naming is hard. So eventually you'll want to rename a [symbol][symbol]. If your choice of editor includes some basic refactoring tools, chances are you can easily trigger a rename refactoring. In VS Code, _the best editor_, you can simply hit F2 with the symbol you want to rename selected. This should then update all references to that symbol to use the new name.
+
+[symbol]: https://en.wikipedia.org/wiki/Symbol_(programming)
+
+## Write less/more code
```

Comment:
What about mentioning the art of _deleting_ code??

Reply:
Great point! Will see if I can bake it into this section or make it its own.